tests/test_execute.py

```diff
@@ -352,7 +347,7 @@ def test_input_default(self):
         """
         my_process = create_complex_proces()
         service = Service(processes=[my_process])
-        self.assertEqual(len(service.processes.keys()), 1)
+        self.assertEqual(len(list(service.processes.keys())), 1)
```

Comment:
could be replaced with `len(service.processes)`

Reply:
done.